src/js/utils/ErrorMessagesUtil.js

```diff
@@ -0,0 +1,116 @@
+import {deepCopy} from './Util';
+
+const TEMPLATE_VAR_REGEXP = /\{([^\}]+)}/g;
+
+const ErrorMessagesUtil = {
+
+  /**
+   * Create an `errorMessage` table that can be passed to `RAMLValidator.clone`
+   * in order to provide custom error messages to RAML validator.
+   *
+   * @param {Object} config - The pre-processed error messages
+   * @returns {Object} Returns the errorMessages object
+   */
+  errorMessagesFromConfig(config) {
+    return Object.keys(config).reduce(function (memo, errorConstant) {
+      const errorMessageTests = config[errorConstant];
+
+      // Create a function override for the RAML validator
+      // NOTE: We are creating a clojure for ensuring the correct
```

Comment:
it is `closure` :) 
and I think the part of the phrase is missing "ensuring the correct..." ?